custom-set.json

```diff
@@ -1,90 +1,54 @@
 {
    "#": [
-      "This problem is not too difficult but is long, with most languages",
-      "requiring a number of set operations.  Most languages test sets of",
-      "ints, which are represented here.  If you want to tests sets of",
-      "strings for example instead, just derive strings from the ints.",
-      "",
-      "Some languages test that integer-valued floats are considered distinct",
-      "from ints or test other types.  To simplify the JSON for test data",
-      "conversion programs these tests with non-int values are collected in",
-      "a separate section.  (Well, I confess, I was finding it awkward to",
-      "deal with this data in Go so I pulled it out.  I'd done the work to",
-      "collect these test cases though and I didn't want to throw them away",
-      "so I put them in that separate section.  If someone writing a test",
-      "data conversion program can use this data and wants to organize it",
-      "differently, go for it.  We'll deal with it.)",
-      "",
-      "Most languages rely on a solver-implemented 'equal' function for",
-      "testing other functions.  Equal tests should come before other",
-      "tests that rely on it.",
-      "",
-      "Languages vary some in terminology, for example in element/member,",
-      "add/put, size/length/cardinality.",
-      "",
-      "Languages often require some functions for getting data in or out",
-      "of sets, such as 'to-list' or 'string' methods.  These are generally",
-      "language specific and so are not represented here.",
-      "",
-      "Some languages with mutable types test functions that modify sets,",
-      "often by assigning variables in a sequence of operations.",
-      "This also is not represented here."
+     "These tests cover the core components of a set data structure: checking",
+     "presence, adding, comparing and basic set operations. Other features",
+     "such as deleting elements, checking size, sorting are not tested, but",
+     "you can add them if they are interesting in your language",
+     "",
+     "Tests about mixed-type sets are not included because the ability",
+     "to implement that will vary by language. If your language supports it",
+     "and you want to implement mixed-type sets, feel free."
    ],
-   "equal": {
-      "description": ["Test two sets for equality."],
+   "empty": {
+     "description": "Returns true if the set contains any elements",
+     "cases": [
+       {
+          "description": "sets with no elements are empty",
+          "set": [],
+          "expected": true
+       },
+       {
+          "description": "sets with elements are not empty",
+          "set": [1],
+          "expected": false
+       }
+     ]
+   },
+   "contains": {
+      "description": ["Sets can report if they contain an element"],
```

Comment:
you have a list of strings here (and it contains only a single element), whereas in `empty` it's just a string. I feel like they should be consistent (but I don't care which of the two you pick)